### DISCUSSION TITLE
Using MQ 135 gas sensor

### DISCUSSION THREAD

**Discussion by triatebr** [BIPES/BIPES#183] [General]
Is it possible to use BIPES together with a gas sensor called MQ135?
Does anyone have examples or tips?
As hardware I'm using ESP8266
thanks

**Comment by jetannenbaum** [comment]
BIPES does not currently support this sensor.  You can find support here, outside of BIPES: https://github.com/rubfi/MQ135.

**Reply by triatebr** [reply]
Thank you for the answer
Would you or someone know how to inform me if BIPES works with a gas sensor?
I want to create a prototype to measure polluting gases.

**Reply by jetannenbaum** [reply]
Unfortunately, I do not own the sensor and I only work with the Raspberry Pi Pico (RP2040) based boards,.     I looked at the library, and it is simply reading from an analog input, then doing some calculations.  There is also a digital output (low - no worries/ high - gas detected).  You probably do not need the library, although it performs the calculations for you.
